src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,111 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.*;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String input = Console.readLine();
+        Map<String, Integer> race = new HashMap<>();
+        try {
+            getNames(race, input);
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(e.getMessage());
+        }
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        int num;
+        try {
+            num = parseNumber(Console.readLine());
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(e.getMessage());
+        }
+
+        System.out.println("실행 결과");
+
+        for (int i = 0; i < num; i++) {
+            for(Map.Entry<String, Integer> entry : race.entrySet()){
+                int r = randomNum();
+                move(entry, goStop(r));
+            }
+            printRace(race);
+        }
+
+        List<String> winners = getWinners(race);
+        printWinner(winners);
+    }
+
+    public static void getNames(Map<String, Integer> race, String input) throws IllegalArgumentException {
+        if (input.isBlank()) {
+            throw new IllegalArgumentException("빈 문자열을 입력할 수 없습니다.");
+        }
```

Comment:
```suggestion
    public static void getNames(Map<String, Integer> race, String input) {
        if (input.isBlank()) {
            throw new IllegalArgumentException("빈 문자열을 입력할 수 없습니다.");
        }
```

언체크 예외는 컴파일 타임에 잡지 않아도 됩니다